packages/contracts/deploy-config/devnetL1-template.json

```diff
@@ -44,6 +44,7 @@
   "validatorManagerTrustedValidator": "0x70997970C51812dc3A010C7d01b50e0d17dc79C8",
   "validatorManagerMinRegisterAmount": "0x32",
   "validatorManagerMinActivateAmount": "0x64",
+  "validatorManagerMptFirstOutputIndex": "0x0",
```

Comment:
```suggestion
  "validatorManagerMptFirstOutputIndex": "0x1",
```

Reply:
Why do you think it has to be 1?

Reply:
Since the MPT transition occurs in block 2 after 4 seconds, shouldn't the first output index after the MPT transition be 0?

Reply:
You're right, I was wrong.
output index = 0 for startingBlockNumber(=0) 
MPT first output index = 1 is correct